xray/services/utils/policybody.go

```diff
@@ -93,6 +108,31 @@ func CreateSeverityPolicyCriteria(minSeverity Severity) *PolicyCriteria {
 	}
 }
 
+func CreateExposuresPolicyCriteria(minSeverity Severity, secrets, applications, services, iac, maliciousCode bool) *PolicyCriteria {
```

Comment:
I can see this function is being used in test_utils.go only. If this is the case we might consider moving it into test_utils.go

Also delete maliciousCode arg since it is not in use